public/locales/en.json

```diff
@@ -185,6 +185,7 @@
         "untitledNote": "Untitled",
         "placeholder": "← Start by entering a title here\n===\nVisit the features page if you don't know what to do.\nHappy hacking :)",
         "invalidYaml": "The yaml-header is invalid. See <0></0> for more information.",
+        "deprecatedTags": "The comma-separated definition of tags in the YAML-metadata is deprecated. Use a YAML-collection instead. For more details take a look at the <0></0>.",
```

Comment:
In other messages we used a "why" link. Please keep this syntax.

Reply:
As you can extract from just the line above this change, we didn't always use "Why?". I'd say that _"For more details take a look at the FAQ."_ is more meaningful than a simple _"Why?"_.

Reply:
Then we should be consistent. And I thought we would use the why. 
Anyway. The "read here for more information" shouldn't be part of every message, because it's a common phrase.

Reply:
And we should adjust the other messages in another pr

Reply:
As we had only one other message where the "Read here for more information" makes sense, I just changed that here.

Reply:
What about the sequence diagram? The PDF export entry?

Reply:
The "Read here for more information" is too long for the PDF-export, therefore it makes more sense to keep the "Why?" there. The sequence-diagram is the other location I mentioned and changed.

Reply:
Right. I forgot to pull :sweat_smile: